.env.example

```diff
@@ -0,0 +1,43 @@
+# Services
+CONVECTION_API_BASE=https://convection-staging.artsy.net/api
+DELTA_API_BASE=http://delta-staging.artsy.net
+DIFFUSION_API_BASE=https://diffusion-staging.artsy.net/api
+EMBEDLY_ENDPOINT=https://i.embed.ly/1/display
+GALAXY_API_BASE=https://artsy-galaxy-staging.herokuapp.com
+GEMINI_API_BASE=https://media.artsy.net
+GEMINI_ENDPOINT=https://d7hftxdivxxvm.cloudfront.net
+GOOGLE_CSE_API_BASE=https://www.googleapis.com/customsearch/v1
+GRAVITY_API_BASE=https://stagingapi.artsy.net/api/v1
+GRAVITY_API_URL=https://stagingapi.artsy.net/
+HMAC_SECRET=https://www.youtube.com/watch?v=DLzxrzFCyOs
+IMPULSE_API_BASE=https://impulse-staging.artsy.net/api
+POSITRON_API_BASE=https://writer-staging.artsy.net/api
+PREDICTION_ENDPOINT=https://live-staging.artsy.net
+REDIS_URL=redis://127.0.0.1:6379
+
+# Keys/Secrets
```

Comment:
We could possibly leave non OSS stuff out of these files now that things are consolidated in `config` anyway.